agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java

```diff
@@ -269,12 +269,18 @@ private static void logConfigurationWarnings(Configuration config) {
               + "If you're on premise, you can use APPLICATIONINSIGHTS_AUTHENTICATION_STRING environment variable to pass the client ID and secret, "
               + "e.g. APPLICATIONINSIGHTS_AUTHENTICATION_STRING=Authorization=AAD;ClientId={CLIENT_ID};ClientSecret={CLIENT_SECRET}.");
     }
-    if (config.sampling.percentage != null && config.sampling.requestsPerSecond != null) {
+    if (config.sampling.requestsPerSecond != null && config.sampling.percentage != null) {
       configurationLogger.warn(
           "Sampling \"requestsPerSecond\" and \"percentage\" should not be used at the same time."
               + " Please remove one of them.");
       config.sampling.percentage = null; // requestsPerSecond takes priority
     }
+    if (config.sampling.requestsPerSecond != null && config.sampling.ingestion != null) {
+      configurationLogger.warn(
+          "Sampling \"requestsPerSecond\" and \"ingestion\" should not be used at the same time."
+              + " Please remove one of them.");
```

Comment:
```suggestion
              + " Only \"requestsPerSecond\" is taken into account. Please remove \"ingestion\" or \"requestsPerSecond\" from your configuration.");
```
Same type of comment for the previous `if`.

Reply:
this code is gone now after rewrite